tests/frontend/compilers/test_xcov.py

```diff
@@ -265,40 +309,6 @@ def test_incorrect_s2gate_modes(self, num_pairs):
         ):
             res = prog.compile("Xcov")
 
-    @pytest.mark.parametrize("num_pairs", [4, 5, 6, 7])
```

Comment:
why could these be deleted?

Reply:
The compilers are no longer in charge of validating parameter values 🙂 These tests have been deleted and moved to `test_program`